### ISSUE TITLE
Throughput on A100/H100?

### ISSUE THREAD

**Issue by jzhang38** [stanford-crfm/levanter#542]
Thanks for such a nice framework! 

Can we have a table of different model sizes' throughput & MFU & HFU on Nvidia GPUs (or is it already there somewhere)? I am always curious how will optimized jax code perform on GPU.

**Comment by dlwh** [comment]
Hi @jzhang38 sorry for being slow. I was on vacation.

It would be nice to have a public table that's continously updated or something like that. I'll share this internal doc that @vadam5 was fleshing out. https://docs.google.com/spreadsheets/d/1H6Ev3FY0d4VLUM6G4igVClIS296pJ7MrSpajZvmVCOQ/edit?usp=sharing

There's something wrong with the 13B runs (the nodes we were using failed before we sorted it out).

We don't have regular access to H100s (yet) so can't really do anything there besides the numbers I've already shared.

We're still honestly mostly a TPU shop, except for smaller, mostly single node runs.
